packages/at_onboarding_cli/lib/src/util/print_full_parser_usage.dart

```diff
@@ -0,0 +1,52 @@
+import 'dart:io';
+
+import 'package:args/args.dart';
+
+extension PrintAllArgParserUsage on ArgParser {
```

Comment:
A little extension to help with pretty printing of the auth_cli's ArgParsers' usage when there are subcommands involved